app.js

```diff
@@ -155,6 +155,7 @@ function createApp(config) {
 
   // catch 404 and forward to error handler
   const requestHandler = (req, res, next) => {
+    logger.info('Error when handling a request', { request: req, response: res })
```

Comment:
Not very sure whether the logger serialized the `{ request: req, response: res}`. If yes, I think it's good. If not, I think app insights will only show `{request: [Object], response: [Object] }`.

Reply:
ty! will check